src/solver/optimisation/opt_appel_solveur_lineaire.cpp

```diff
@@ -446,25 +677,51 @@ void OPT_EcrireJeuDeDonneesLineaireAuFormatMPS(void* Prob, uint numSpace, char T
     int ExistenceDUneSolution;
     double* X;
 
-    Probleme = (PROBLEME_SIMPLEXE*)Prob;
-
-    ExistenceDUneSolution = Probleme->ExistenceDUneSolution;
-    if (ExistenceDUneSolution == OUI_SPX)
-        ExistenceDUneSolution = OUI_ANTARES;
-
-    NombreDeVariables = Probleme->NombreDeVariables;
-    TypeDeBorneDeLaVariable = Probleme->TypeDeVariable;
-    Xmax = Probleme->Xmax;
-    Xmin = Probleme->Xmin;
-    X = Probleme->X;
-    CoutLineaire = Probleme->CoutLineaire;
-    NombreDeContraintes = Probleme->NombreDeContraintes;
-    SecondMembre = Probleme->SecondMembre;
-    Sens = Probleme->Sens;
-    IndicesDebutDeLigne = Probleme->IndicesDebutDeLigne;
-    NombreDeTermesDesLignes = Probleme->NombreDeTermesDesLignes;
-    CoefficientsDeLaMatriceDesContraintes = Probleme->CoefficientsDeLaMatriceDesContraintes;
-    IndicesColonnes = Probleme->IndicesColonnes;
+    if (Type == ANTARES_SIMPLEXE)
+    {
+        Probleme = (PROBLEME_SIMPLEXE*)Prob;
+
+        ExistenceDUneSolution = Probleme->ExistenceDUneSolution;
+        if (ExistenceDUneSolution == OUI_SPX)
+            ExistenceDUneSolution = OUI_ANTARES;
+
+        NombreDeVariables = Probleme->NombreDeVariables;
+        TypeDeBorneDeLaVariable = Probleme->TypeDeVariable;
+        Xmax = Probleme->Xmax;
+        Xmin = Probleme->Xmin;
+        X = Probleme->X;
+        CoutLineaire = Probleme->CoutLineaire;
+        NombreDeContraintes = Probleme->NombreDeContraintes;
+        SecondMembre = Probleme->SecondMembre;
+        Sens = Probleme->Sens;
+        IndicesDebutDeLigne = Probleme->IndicesDebutDeLigne;
+        NombreDeTermesDesLignes = Probleme->NombreDeTermesDesLignes;
+        CoefficientsDeLaMatriceDesContraintes = Probleme->CoefficientsDeLaMatriceDesContraintes;
+        IndicesColonnes = Probleme->IndicesColonnes;
+    }
+    else
+    {
+        ProblemePourPne = (PROBLEME_A_RESOUDRE*)Prob;
+
+        ExistenceDUneSolution = ProblemePourPne->ExistenceDUneSolution;
+        if (ExistenceDUneSolution == SOLUTION_OPTIMALE_TROUVEE)
+            ExistenceDUneSolution = OUI_ANTARES;
+
+        NombreDeVariables = ProblemePourPne->NombreDeVariables;
+        TypeDeBorneDeLaVariable = ProblemePourPne->TypeDeBorneDeLaVariable;
+        Xmax = ProblemePourPne->Xmax;
+        Xmin = ProblemePourPne->Xmin;
+        X = ProblemePourPne->X;
+        CoutLineaire = ProblemePourPne->CoutLineaire;
+        NombreDeContraintes = ProblemePourPne->NombreDeContraintes;
+        SecondMembre = ProblemePourPne->SecondMembre;
```

Comment:
PNE has been removed from the develop branch, this distinction is no longer necessary.